scalafmt-tests/src/test/resources/newlines/source_keep.stat

```diff
@@ -521,3 +522,386 @@ val deployment = system
+      in.readObject
+    }
+  ) ++ 2
+}
+>>>
+object a {
+  val a = (intercept[java.lang.IllegalStateException] {
+    in.readObject
+  }) ++ 2
+}
+<<< 3.20 enclosed, assignment, infix on right, narrower
+maxColumn = 60
+danglingParentheses = false
+===
+object a {
+  val a = 2 ++ (
+    intercept[java.lang.IllegalStateException] {
+      in.readObject    }  )
+}
+>>>
+object a {
+  val a = 2 ++ (
+    intercept[java.lang.IllegalStateException] {
+      in.readObject
+    })
+}
+<<< 3.21 enclosed, assignment, infix in middle, narrower
+maxColumn = 60
+danglingParentheses = false
+===
+object a {
+  val a = 2 ++ (
+    intercept[java.lang.IllegalStateException] { in.readObject
+    }  ) ++ 3
+}
+>>>
+object a {
+  val a = 2 ++ (
+    intercept[java.lang.IllegalStateException] {
+      in.readObject
+    }) ++ 3
+}
+<<< 3.22 enclosed, assignment, by itself, narrow
+maxColumn = 40
+danglingParentheses = false
+===
+object a {
+  val a = (intercept[java.lang.IllegalStateException] {
+      in.readObject
+    }
+  )
+}
+>>>
+object a {
+  val a = (intercept[
+    java.lang.IllegalStateException] {
+    in.readObject
+  })
+}
+<<< 3.23 enclosed, assignment, infix on left, narrow
+maxColumn = 40
+danglingParentheses = false
+===
+object a {
+  val a = (
+    intercept[java.lang.IllegalStateException] {
+      in.readObject
+    }) ++ 2
+}
+>>>
+object a {
+  val a = (
+    intercept[
+      java.lang.IllegalStateException] {
+      in.readObject
+    }) ++ 2
+}
+<<< 3.24 enclosed, assignment, infix on right, narrow
+maxColumn = 40
+danglingParentheses = false
+===
+object a {
+  val a = 2 ++ (intercept[java.lang.IllegalStateException] { in.readObject }
+  )
+}
+>>>
+object a {
+  val a = 2 ++ (intercept[
+    java.lang.IllegalStateException] {
+    in.readObject
+  })
+}
+<<< 3.25 enclosed, assignment, infix in middle, narrow
+maxColumn = 40
+danglingParentheses = false
+===
+object a {
+  val a = 2 ++ (intercept[java.lang.IllegalStateException] {
+    in.readObject
+  }) ++ 3
+}
+>>>
+object a {
+  val a = 2 ++ (intercept[
+    java.lang.IllegalStateException] {
+    in.readObject
+  }) ++ 3
+}
+<<< 3.26 enclosed, def, chain follows
+maxColumn = 80
+===
+  def activationFutureFor(endpoint: ActorRef)(implicit
+      timeout: Timeout,
+      executor: ExecutionContext): Future[ActorRef] =
+    (supervisor
+      .ask(AwaitActivation(endpoint))(timeout))
+      .map[ActorRef]({
+        case EndpointActivated(`endpoint`) ⇒ endpoint
+        case EndpointFailedToActivate(`endpoint`, cause) ⇒ throw cause
+      })
+>>>
+def activationFutureFor(
+    endpoint: ActorRef
+)(implicit timeout: Timeout, executor: ExecutionContext): Future[ActorRef] =
+  (supervisor
+    .ask(AwaitActivation(endpoint))(timeout))
+    .map[ActorRef]({
+      case EndpointActivated(`endpoint`) ⇒ endpoint
+      case EndpointFailedToActivate(`endpoint`, cause) ⇒ throw cause
+    })
+<<< SKIP 3.27 enclosed in block, def, chain follows
```

Comment:
Do you plan to remove `SKIP` in the next PRs before release?

Reply:
yes, it needs a matching change handling unfold in select chains